django/db/backends/postgresql/base.py

```diff
@@ -149,6 +150,12 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     # PostgreSQL backend-specific attributes.
     _named_cursor_idx = 0
 
+    def get_database_version(self):
+        # The version is formed by converting the major, minor, and revision
```

Comment:
This comment is a bit confusing since it describes how PostgreSQL returns its version number rather than what this method does.